transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Documents/Account.java

```diff
@@ -0,0 +1,28 @@
+package com.transcriptanalyzer.transcript.User_Requests_Intents.Documents;
+
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document
+public class Account {
```

Comment:
Maybe add a brief description about what this class represents and why/how we use it.